inception-project/inception
# #1875 - Logout from external authentication

**What's in the PR**
- Add property to configure logout URL
- Update documentation

**How to test manually**
* Check that login works
* Log into INCEpTION  Community, set `auth.logoutUrl=https://sp.ukp.informatik.tu-darmstadt.de/Shibboleth.sso/Logout`, start INCEpTION  locally, log in and log out and see that you are logged out in INCEpTION Community

**Automatic testing**

**Documentation**
